diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/DiktatRuleSetProvider.kt

```diff
@@ -163,10 +162,33 @@ class DiktatRuleSetProvider(private var diktatConfigFile: String = DIKTAT_ANALYS
 
     private fun validate(config: RulesConfig) =
             require(config.name == DIKTAT_COMMON || config.name in Warnings.names) {
-                val closestMatch = Warnings.names.minBy { Levenshtein.distance(it, config.name) }
+                val closestMatch = Warnings.names.minByOrNull { Levenshtein.distance(it, config.name) }
                 "Warning name <${config.name}> in configuration file is invalid, did you mean <$closestMatch>?"
```

Comment:
Does it happen in case if config is empty? If so do we need this warning message?

Reply:
Config can't be empty, at the very least we will use config included in the jar.

Reply:
Oh you used `minByOrNull` because minBy is deprecated